examples/node/index.ts

```diff
@@ -49,7 +49,7 @@ async function sendPingToRandomNodeInCluster() {
         clientName: "test_cluster_client",
     });
     // The empty array signifies that there are no additional arguments.
-    const pong = await client.customCommand(["PING"], "randomNode");
+    const pong = await client.customCommand(["PING"], { route: "randomNode" });
```

Comment:
can you update the dependency in the package.json to 1.1.0
https://github.com/valkey-io/valkey-glide/blob/77e677b610b2ca93c8138dcb74e9f4f16597f52e/examples/node/package.json#L4